source/fe/mapping_fe.cc

```diff
@@ -923,9 +923,8 @@ MappingFE<dim, spacedim>::MappingFE(const FiniteElement<dim, spacedim> &fe)
   : fe(fe.clone())
   , polynomial_degree(fe.tensor_degree())
 {
-  Assert(polynomial_degree >= 1,
-         ExcMessage("It only makes sense to create polynomial mappings "
-                    "with a polynomial degree greater or equal to one."));
+  AssertDimension(fe.tensor_degree(), 1);
```

Comment:
Can you leave this assert there and add ``Assert(fe.tensor_degree() == 1, ExcNotImplemented())``?